feat: add release notes for timeouts (#3517)
gitpod-io/website: src/lib/contents/changelog/timeouts.md

---

> title: Flexible Timeouts

Author: ```suggestion
title: Introducing: Flexible Timeouts 🥳  !
```

excerpt: You can now set your workspace timeout up to 24 hours using the command line, direct in your favorite editor or IDE or configuring a global default.
date: 2023-03-15
image: timeouts.jpg
ogImage: timeouts.jpg
alt: Input box for timeouts set in a users preferences page.
---

<script>
  import Contributors from "$lib/components/changelog/contributors.svelte";
</script>

> You can now set your workspace timeout up to 24 hours using the command line, direct in your favorite editor or IDE or configuring a global default! Setting a configurable timeout is useful if you're working on a long running task, need to leave your workspace open for a while and then revisit it, or whilst collaborating or sharing previews with others.

Reviewer: ```suggestion
You can now set your workspace timeout up to 24 hours using the command line, direct in your favorite editor or IDE or configure a global default! Setting a configurable timeout is useful if you're working on a long running task, need to leave your workspace open for a while and then revisit it, or whilst collaborating or sharing previews with others.
```

> To try out this feature, simply type: `gp timeout set 1h` in your running workspace to set the timeout for the currently running workspace. Or, go to your [user preferences](https://gitpod.io/user/preferences) to set the default value for all future timeouts.

Reviewer: Maybe we can be explicit about what this command does? I don't really like the wording in my recommendation but just to illustrate what I am looking for.

```suggestion
To try out this feature, simply type: `gp timeout set 1h` in your running workspace to set the timeout for the currently running workspace to 1 hour. Or, go to your [user preferences](https://gitpod.io/user/preferences) to set the default value for all future timeouts.
```

Author: ```suggestion
To try out this feature, type: `gp timeout set 1h` to set the timeout for the currently running workspace to 1 hour. Or, go to your [user preferences](https://gitpod.io/user/preferences) to set the default value for all future timeouts.
```
